tests/conftest.py

```
@@ -0,0 +1,79 @@
+def client(app):
+    return TestClient(app, raise_server_exceptions=False)
+
+@pytest.fixture(scope="session")
+def dynamodb_table():
+    import boto3
+    dynamodb = boto3.resource(
+        "dynamodb",
+        region_name="ap-northeast-1",
+        endpoint_url="http://localhost:4566",
+        aws_access_key_id="test",
+        aws_secret_access_key="test",
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to argument: "aws_secret_access_key"

(S106)